### PR TITLE
virt: introduce add qemu options support

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
ID: 1236501

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -726,6 +726,33 @@ def add_nographic(devices):
...
+            """
+            Add qemu option, such as '-msg timestamp=on|off'
+
+            :param devices: qcontainer object
+            :param name: string type option name
+            :param optsinfo: list like [(key, val, vtype)]
+            """
+            if devices.has_option(name):
+                options = []
+                for info in optsinfo:
```
Comment (suqinhuang) [inline virttest/qemu_vm.py]: Hi xu,
Do you mean options here?

**Comment by suqinhuang** [comment]
LGTM
acked-by: Suqin Huang

**Comment by spcui** [comment]
Acked-by: scui@redhat.com

**Comment by xutian** [comment]
thanks @suqinhuang and @spcui

**Pull request merged**